src/config.json

```diff
@@ -5,6 +5,6 @@
         "email_password": "super_secure_password",
         "recipient_emails": ["homeowner@example.com"]
     },
-    "mac_addrs": ["AA:BB:CC:DD:EE:FF"],
+    "mac_addrs": ["A0:9F:10:1B:D7:05", "AC:83:F3:BC:A7:61", "C4:DD:57:13:A7:FF", "EC:FA:BC:55:6E:A2", "D4:A6:51:40:1D:69", "D4:A6:51:40:34:F0", "10:52:1C:4C:54:9E"],
```

Comment:
We shouldn't be installed the MAC addresses with the package. This is fine for getting the prototype up and running, however, we do need to find the best way to allow the end user to select the IoT devices they want monitored.

Reply:
I agree that it should ship with this array being empty, but the user will fill it themselves and the code will pull the mac_addrs from here. So for testing purposes, we need this file to have real data. Won't they add it through the dashboard?

Reply:
^ They MAC addresses should be added through the dashboard, but I agree that having pre-filled information is better for testing.